packages/hs-react-ui/src/components/Dropdown/Dropdown.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+import { storiesOf, addDecorator } from '@storybook/react';
+import { boolean, number } from '@storybook/addon-knobs';
+import { action } from '@storybook/addon-actions';
+import { withDesign } from 'storybook-addon-designs';
+import { Dropdown } from './Dropdown';
+
+addDecorator(withDesign);
+const design = {
+  type: 'figma',
+  url: 'https://www.figma.com/file/3r2G00brulOwr9j7F6JF59/Generic-UI-Style?node-id=102%3A28',
+};
+
+storiesOf('Dropdown', module).add(
+  'Basic',
+  () => {
+    const options = ['Apple', 'Banana', 'Grapefruit', 'Grape', 'Orange', 'Watermelon'];
+    return (
+      <>
+        <label htmlFor="fave-fruit">What's your favorite fruit?</label>
+        <Dropdown
+          clearable={boolean('clearable', false)}
+          elevation={number('elevation', 2, { range: true, min: 0, max: 5, step: 1 })}
+          multi={boolean('multi', false)}
+          name="fave-fruit"
+          onBlur={action('onBlur')}
+          onClear={action('onClear')}
+          onSelect={action('onSelect')}
+          options={options}
```

Comment:
should we include a `values` knob? It'd just be a select with the values - and we wouldn't be able show multiselects properly but idk. May be better to put into the docs

Reply:
I've included one (doesn't work for multi-select), but yeah we'll definitely want to put this in the docs anyway.